deploy.sh

```diff
@@ -16,8 +16,20 @@ region=${region:-us-east-1}
 echo "ENV: $env"
 echo "Region: $region"
 
-# Download model artifacts
-dvc pull
+shift $((OPTIND - 1))
 
-# Build container and deploy functions
-serverless deploy --stage "${env}" --region "${region}"
+# Check if a script name was provided as a command line argument
+if [ -z "$1" ]; then
+    echo "Error: Not sure whether to run cloudformation or serverless"
+    exit 1
+fi
+
+if [ "$1" == "cf" ]; then
+    aws cloudformation deploy --template-file cloudformation.yml --stack-name poprox-research-recommender-"${env}" --parameter-overrides env="${env}" --region "${region}" --capabilities CAPABILITY_NAMED_IAM
+elif [ "$1" == "sls" ]; then
+    # Download model artifacts
+    dvc pull
```

Comment:
I mad the appropriate change also over in #132, but this can be `dvc pull -R models` to avoid pulling evaluation data that isn't necessary for image deployment.